src/common/components/Icon/Icon.styles.tsx

```diff
@@ -0,0 +1,30 @@
+import styled, { css } from 'styled-components'
```

Comment:
Please don't use styled-components.

Reply:
@muhadovic I think this Icon uses in `input` global component.